src/multibody/geometry.hpp

```diff
@@ -274,8 +287,15 @@ namespace se3
     
     ///
     /// \brief Check if at least one of the collision pairs has its two collision objects in collision.
+    ///        The results are stored in the vector GeometryData::collision_results.
+    ///
+    /// \param[out] the index of the colliding pair if the function returns true.
     ///
-    bool isColliding() const;
+    bool isColliding(Index& collidingPair);
```

Comment:
From a design point of view, adding a field to `GeometryData` for this seems to me a bit too much. But it is definitely possible.